test/test_ssl/test_dhparam.yml

```diff
@@ -0,0 +1,16 @@
+web5:
+  image: web
+  expose:
+    - "85"
+  environment:
+    WEB_PORTS: "85"
+    VIRTUAL_HOST: "web5.nginx-proxy.tld"
+    HTTPS_METHOD: nohttp
```

Comment:
`HTTPS_METHOD` does not seem to have any effect on the outcome of the test. I would remove it